content/en/account_management/org_settings/ip_allowlist.md

```diff
@@ -0,0 +1,88 @@
+---
+title: IP Allowlist
+kind: documentation
+---
+
+{{< callout url="/help/" header="Get Started with IP Allowlist" >}}
+The IP allowlist feature is available for enterprise-level customers only. Request access by contacting support.
+{{< /callout >}}
+
+## Overview
+
+The IP allowlist controls which networks can be used to access your data in Datadog. By limiting allowed networks, you can protect your resources from data exfiltration and insider threats.
+
+When the IP allowlist is enabled, only IP addresses or CIDR ranges in the allowlist can access the Datadog API and UI. 
+
+### Blocked and allowed resources
+
+If a user's IP is not contained in the IP allowlist, they are effectively blocked from accessing and using:
+
+- Datadog’s web UI
+- Datadog’s public [API][1], including both documented and unpublished endpoints
+- Datadog’s mobile apps (iOS, Android)
+- Third-party applications that access Datadog through OAuth
+
+The IP allowlist feature does not block access to the following:
+- Data ingest endpoints to which the Agent sends data, such as metrics, traces, and logs
+- The [Validate API key][2] endpoint, which is often used by data ingestion
```

Comment:
"which is often used by data ingestion"

I might word differently, something like "which is used internally by Datadog Agents before submitting data". As written, it kind of sounds like it's used by Datadog's server-side ingestion system.

Reply:
Yeah this is not a major use, so can also consider removing that line completely (since it's kind of covered by the first data ingest line)